### ISSUE TITLE
[RFC] Add a list view to the dashboard

### ISSUE THREAD

**Issue by josebolos** [pa11y/pa11y-dashboard#165]
We have a number of concerns regarding how the information is being displayed in the dashboard. There are also a few related issues that have been created over time, e.g. #106 

We're thinking the display of information could be improved by using a simple list instead of the current 3 column layout.

This is a quick mockup made with the developer tools displaying the same information:

![image](https://cloud.githubusercontent.com/assets/1792637/20928652/c00e7a18-bbbe-11e6-9135-8eb15f586e54.png)

(The WCAG2AA label is missing, that would have to stay).

The area needed for display would be slightly bigger, but as a result we would getmore horizontal space to display the full name of the page and, ideally, the URL too (as shown in the image above).

It would also make easier to add future improvements, like for example:
* Adding either `categories`, `sites` or `labels`, which would avoid having to manually add such data to the site name (as shown with the `[Springer Nature]` text in the picture).
* Possibility of filtering and/or ordering the sites by category/site/label, number of errors, etc.

This could be added in addition to the current view, with maybe the possibility of switching between one and the other (at least for as long as we can maintain both).

**Comment by adamki** [comment]
Hi there. Looking to start with something small(reference issue [106](https://github.com/pa11y/dashboard/issues/106)). pa11y seems pretty great and I would love to help out where possible. 

For the time being, I suppose I can proceed with: 

```
The minimum useful pull request would include CSS to create the list version of the tiles, and a button (or something) somewhere on the page that lets people switch between the tile and list views by setting a cookie. I think we wanted to default to the list view.
```

as described by @hollsk in the above mentioned issue. However, any more direction or specification would be appreciated.

Also...Is there any communication channel?

**Comment by hollsk** [comment]
We're looking forward to your contribution! You can get an invite to our Slack channel here: https://pa11y-slack.herokuapp.com/ - it might be a little slow at the moment because of the holidays though, so bear with us.

**Comment by josebolos** [comment]
Fixed in #279.
